src/command/PlayerCommands.cxx

```diff
@@ -129,7 +129,7 @@ handle_status(Client &client, [[maybe_unused]] Request args, Response &r)
 	      COMMAND_STATUS_PLAYLIST ": {}\n"
 	      COMMAND_STATUS_PLAYLIST_LENGTH ": {}\n"
 	      COMMAND_STATUS_MIXRAMPDB ": {}\n"
-	      COMMAND_STATUS_STATE ": {}\n",
```

Comment:
Which commit introduced this bug? This should be mentioned in the commit message so we know whether a backport is necessary.